hooks/pre-commit

```diff
@@ -2,7 +2,7 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-./hooks/apply-git-secrets.sh
+eval "$( cd "$(dirname "$0")" ; pwd -P )/apply-git-secrets.sh"
```

Comment:
FYI my usual way of getting the git root is `git rev-parse --show-toplevel`, might be slightly simpler here.